src/mds/MDSRank.cc

```diff
@@ -3350,6 +3350,8 @@ void MDSRank::create_logger()
                             "exi", PerfCountersBuilder::PRIO_INTERESTING);
     mds_plb.add_u64_counter(l_mds_imported_inodes, "imported_inodes", "Imported inodes",
                             "imi", PerfCountersBuilder::PRIO_INTERESTING);
+    mds_plb.add_u64_counter(l_mds_slow_reply, "slow_reply", "Slow replies", "slr",
```

Comment:
`slow reply' is a bit confusing - how about using `slow requests'?

Reply:
IMO  l_mds_request and l_mds_reply are not necessarily equal, so named it as l_md_slow_reply which is more related to l_mds_reply. l_mds_slow_request and l_mds_slow_reply both works for me. If you prefer l_mds_slow_request I will change to it.

Reply:
Ah ok - let's keep it as `l_mds_slow_reply` then. thx!